### PR TITLE
Bugfix/typos and grammar

### PR DESCRIPTION
**Pull request by oliverrahner**

### CODE REVIEW

File: docs/integration/build-message.adoc [hunk 1]
```diff
@@ -200,24 +200,24 @@ The structure of the payload depends on the technical message type. Its always s
...
+Envelope and content are packaged into one container by using the technique of “Delimited Messages”. Please note that this is *not* simply copying both memory buffers into one buffer. Please refer to https://developers.google.com/protocol-buffers/docs/techniques[https://developers.google.com/protocol-buffers/docs/techniques#streaming]
 
 [NOTE]
 ====
 Note that this concept is not supported in all protobuf libraries (in Java and node.js it is, in C++ it is not in every version)
 ====
 
-If building streaming is required for the language and libraries, you use, Note, that Delemited messages are attached to each other like this: Length1,Content1,Length2,Content2,…. The variable size of Length is the length of a varint; see https://developers.google.com/protocol-buffers/docs/encoding#varints.
+If building streaming is required for the language and libraries that you use, please note that Delimited messages are attached to each other like this: Length1,Content1,Length2,Content2,…. The variable size of Length is the length of a varint; see https://developers.google.com/protocol-buffers/docs/encoding#varints.
```
Comment (Frank-Wiebeler) [inline docs/integration/build-message.adoc:210]: minor d here?

File: docs/integration/reonboarding.adoc [hunk 2]
```diff
@@ -14,17 +14,17 @@ Internally, the endpoint gets the same identification (deviceAlternateId, sensor
 
 [IMPORTANT]
 ====
-Reonboarding is major as certificates are only valid for currently *one year*
+Reonboarding is very important as certificates are only valid for currently *one year*
```
Comment (Frank-Wiebeler) [inline docs/integration/reonboarding.adoc:17]: By "major" I mean "major due to certification". But we need to update this as reonboarding is only major to those endpoints that do not use router devices.

File: docs/integration/revoke.adoc [hunk 3]
```diff
@@ -54,7 +54,7 @@ The body includes a JSON object:
 |===============================================================================
 |# |Name |Type |Description
 |1 |accountId |String |The ID of the users account
-|2 |endpointIds |Array of String |An Array of endpointIDs, that shall be deleted.
+|2 |endpointIds |Array of String |An Array of endpointIDs that shall be deleted.
```
Comment (Frank-Wiebeler) [inline docs/integration/revoke.adoc:57]: second endpointIDs should be endpointIds

**Review by Frank-Wiebeler — CHANGES_REQUESTED** [review]
Great job, while reviewing I found a few minor issues that might be worth fixing as well within this PR.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  "spellright.language": [
+    "en"
+  ],
+  "spellright.documentTypes": [
+    "markdown",
+    "latex",
+    "plaintext",
+    "asciidoc"
+  ]
+}
```
[the remark this answers is not in the document]
Comment (oliverrahner) [reply .vscode/settings.json:13]: I'm a bit split on this question. I'd really love to have projects behave the same way for everybody using the same IDE... on the other hand, we don't (and can't) assume which IDE should be used.
This general decision would also include include the custom spelling dictionary, which is vital for a documentation project :-/

I'd vote for "keep"...

File: docs/communication.adoc [hunk 5]
```diff
@@ -6,20 +6,20 @@
...
+Communication with the agrirouter includes communication with inbox AND outbox, regardless whether an app only sends messages or only consumes messages. Every message or command is sent to the inbox, results are always delivered through the outbox.
 
 == Prerequisites for communication
 
-Every communication is introduced by the App Instance. The agrirouter endpoint will never contact the app instance.
+Every communication is initiated by the App Instance. The agrirouter endpoint will never contact the app instance.
 
-After application registration, the communication is secured using the SSL certificate retrieved during the onboarding process. The agrirouter provides PK12 certificates as well as PEM certificates which can be set within the SSL context of the request to ensure authenticated communication with the agrirouter.
+After application registration, the communication is secured using the SSL certificate retrieved during the onboarding process. The agrirouter provides PKCS12 certificates as well as PEM certificates which can be set within the SSL context of the request to ensure authenticated communication with the agrirouter.
```
[the remark this answers is not in the document]
Comment (oliverrahner) [reply docs/communication.adoc:15]: will fix

File: docs/integration/reonboarding.adoc (hunk 2 above)
Comment (oliverrahner) [reply docs/integration/reonboarding.adoc:17]: Hmm... I think I don't get what you want to say... what is the German term for what you want to express?

File: docs/integration/revoke.adoc (hunk 3 above)
Comment (oliverrahner) [reply docs/integration/revoke.adoc]: will fix

File: .vscode/settings.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (Frank-Wiebeler) [reply .vscode/settings.json:13]: You're right on the dictionary, so we can keep the file.

File: docs/integration/reonboarding.adoc (hunk 2 above)
Comment (Frank-Wiebeler) [reply docs/integration/reonboarding.adoc:17]: I think "required for CUs" is the correct term. German word would be something like "zwingend erforderlich".

Comment (Frank-Wiebeler) [reply docs/integration/reonboarding.adoc]: "It is vital when not using RouterDevices ..." and I'm fine ;)

Comment (oliverrahner) [reply docs/integration/reonboarding.adoc]: Done

**Review by Frank-Wiebeler — APPROVED** [review]
Well done 🥇

**Pull request merged**